src/Model/ROI.py

```diff
@@ -46,6 +52,105 @@ def delete_roi(rtss, roi_name):
     return rtss
 
 
+def create_roi(rtss, roi_name, roi_coordinates, data_set):
+    """
+        Create new ROI to rtss
+
+        :param rtss: dataset of RTSS
+        :param roi_name: ROIName
+        :param roi_coordinates: Coordinates of pixels for new ROI
+        :param data_set: Data Set of selected DICOM image file
+        :return: rtss, with added ROI
+        """
+
+    number_of_contour_points = len(roi_coordinates) / 3
+    referenced_sop_class_uid = ""
+    referenced_sop_instance_uid = ""
+
+    # Optional Tag
+    if data_set.get("ReferencedImageSequence"):
+        referenced_sop_class_uid = data_set.ReferencedImageSequence[0].ReferencedSOPClassUID
+        referenced_sop_instance_uid = data_set.ReferencedImageSequence[0].ReferencedSOPInstanceUID
+
+    referenced_frame_of_reference_uid = rtss["StructureSetROISequence"].value[0].ReferencedFrameOfReferenceUID
+    roi_number = rtss["StructureSetROISequence"].value[-1].ROINumber+1
+
+    # Colour TBC
+    red = random.randint(0,255)
+    green = random.randint(0,255)
+    blue = random.randint(0,255)
+    rgb = [red, green, blue]
```

Comment:
even though this is randomized, there is a risk of having the same random colour used for multiple ROIs unless one makes sure that the particular rgb isn't used multiple times.